tests/syft/lib/python/string/string_utils_test.py

```diff
@@ -1343,7 +1343,7 @@ def test_formatting_c_limits(self):
     def test_floatformatting(self):
         # float formatting
         for prec in range(100):
-            format = "%%.%if" % prec
```

Comment:
I think this is there to test the % __mod__ infix operator for string formatting which is still allowed and is valid in Python 3. These tests are actually from the Python code base so we try to only change superficial aspects where possible to ensure we dont accidentally miss something.

Reply:
@madhavajay the change is also a valid syntax and  was the result of running black on the files. The change won't break anything for Python 3.

Reply:
We are trying not to change the tests from the official python repo as much as possible which means ignoring formatting / linting issues with the copied tests since the originals have these issues in them already. In this case the % mod infix operator is also being tested so it might be best to leave this test as is.